airbyte-db/src/main/java/io/airbyte/db/instance/configs/ConfigsDatabaseInstance.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020 Airbyte
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.airbyte.db.instance.configs;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.airbyte.commons.resources.MoreResources;
+import io.airbyte.db.Database;
+import io.airbyte.db.instance.BaseDatabaseInstance;
+import io.airbyte.db.instance.DatabaseInstance;
+import java.io.IOException;
+import java.util.function.Function;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ConfigsDatabaseInstance extends BaseDatabaseInstance implements DatabaseInstance {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(ConfigsDatabaseInstance.class);
+
+  private static final String DATABASE_NAME = "configs";
```

Comment:
would it make sense to call this `airbyte_configs`. at this point we might be creating a database in a postgres instance that has other dbs, so `configs` seems somewhat higher probability of having a naming collision than if we prefix it. It also just makes it a bit clearer to anyone exploring the db why that database is for.

Reply:
This name is only used for logging purpose. The real database name is specified in the `.env`. But I can still update it.